package.json

```diff
@@ -29,6 +29,7 @@
     "prepare": "npm run build"
   },
   "dependencies": {
+    "@snyk/dep-graph": "1.19.4",
```

Comment:
Should there be a corresponding change in `package-lock.json`?

Reply:
There would be if we checked in a package-lock.json. We don't check it in for libraries

Reply:
Ah makes sense :)